### PR TITLE
Add runlogwatch.sh entry point to dump ramdisk inspection logs

### PR DESCRIPTION
**Pull request by akiselev1**
This is ironic-inspector-image change in series of PRs implementing this design proposal: https://github.com/metal3-io/metal3-docs/blob/master/design/ironic-debuggability-improvement.md

### CODE REVIEW

File: runlogwatch.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+
+# Ramdisk logs path
+LOG_DIR="/shared/log/ironic-inspector/ramdisk"
+
+while :
+do
+    until  ls "${LOG_DIR}"/*.tar.gz 1> /dev/null 2>&1
+    do
+        sleep 1
```
Comment (dhellmann) [inline runlogwatch.sh]: This was 5 in the other implementation.

Comment (akiselev1) [reply runlogwatch.sh]: Fixed.

**Review by elfosardo — APPROVED** [review]

**Pull request merged**